### PR TITLE
Change prerequisites for installation tutorial

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

**Comment by openpublishbuild** [comment]
Docs Build status updates of commit _[b1b33fa](https://github.com/andxu/azure-docs-sdk-java/pull/852/commits/b1b33fa3d9c04358073520241f64cb8fca941d8c)_: 

:clock10: **Full build**: the average full build time is _70 min(s) 57 sec(s)_, this is based on the last _252_ build(s) for this repository.

**Comment by openpublishbuild** [comment]
Docs Build status updates of commit _[b1b33fa](https://github.com/andxu/azure-docs-sdk-java/pull/852/commits/b1b33fa3d9c04358073520241f64cb8fca941d8c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md](https://github.com/andxu/azure-docs-sdk-java/blob/andxu_install_intellj/docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation?view=azure-java-stable&branch=pr-en-us-852) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation?view=azure-java-preview&branch=pr-en-us-852) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation?view=azure-java-legacy&branch=pr-en-us-852) (azure-java-legacy) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C21%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201905210313269929-852%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gN8Lhz%2BfjoWCIye1VpXk9lPuZH2Gvee576R5X8Yh1%2B0%3D&st=2019-05-21T04%3A33%3A29Z&se=2019-06-21T04%3A38%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by andxu** [comment]
#sign-off

File: docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md [hunk 1]
```diff
@@ -30,7 +30,30 @@ The Azure Toolkit for IntelliJ provides templates and functionality that allow y
 
 There are two methods of installing the Azure Toolkit for IntelliJ: by using the **Settings** dialog box, and by using the **Configure** menu on the start screen. Both installation methods will be demonstrated in the following steps.
 
-[!INCLUDE [azure-toolkit-for-IntelliJ-prerequisites](../includes/azure-toolkit-for-intellij-prerequisites.md)]
+## Prerequisites
+
+The Azure Toolkit for IntelliJ requires to the following software components:
+
+* An [Azure supported Java Development Kit (JDK)](https://aka.ms/azure-jdks)
```
Comment (brunoborges) [inline docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md]: The Azul Zulu JDK shouldn't be a prerequisite for IntelliJ. Developers can use any JDK for development purpose.

Comment (andxu) [reply docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md]: replaced with openjdk and oracle links.

**Comment by openpublishbuild** [comment]
Docs Build status updates of commit _[f54eb24](https://github.com/andxu/azure-docs-sdk-java/pull/852/commits/f54eb24315f8cc2020d37478cd80c01424457a41)_: 

:clock10: **Full build**: the average full build time is _71 min(s) 6 sec(s)_, this is based on the last _262_ build(s) for this repository.

**Comment by openpublishbuild** [comment]
Docs Build status updates of commit _[f54eb24](https://github.com/andxu/azure-docs-sdk-java/pull/852/commits/f54eb24315f8cc2020d37478cd80c01424457a41)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md](https://github.com/andxu/azure-docs-sdk-java/blob/andxu_install_intellj/docs-ref-conceptual/intellij/azure-toolkit-for-intellij-installation.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation?view=azure-java-stable&branch=pr-en-us-852) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation?view=azure-java-preview&branch=pr-en-us-852) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-installation?view=azure-java-legacy&branch=pr-en-us-852) (azure-java-legacy) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201905230059130578-852%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rP3dRQVfATy8nTUBo8eq%2FFWfjfICCixVblAorEg8YCE%3D&st=2019-05-23T02%3A17%3A35Z&se=2019-06-23T02%3A22%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**